### ISSUE TITLE
Banding in rendered images using HybridRenderer

### ISSUE THREAD

**Issue by tbmiller-astro** [pysersic/pysersic#56]
Both Imad and I have noticed banding in the rendered images when using the Fourier space based renderers. This is almost certainly an artifact of the rendering or convolution that is done in Fourier space as that is known to produce these periodic artifacts. The good news is it doesn't appear to affect the fit at all, the recovered properties appear reasonable.

It is unclear exactly what conditions trigger this. It is seen in both ground and space based data. 

Would be good to track down and fix any potential numerical artifacts that are causing this.

**Comment by prappleizer** [comment]
Logging a few examples of the effect here, will begin looking at whether there are any particular identifiable features in the data that seem to correlate. We have a sample of ~7000 galaxies to go through to put together a more "robust" sample of where this kind of thing is happening.  

<img width="1327" alt="Screenshot 2024-06-06 at 12 50 06 PM" src="https://github.com/pysersic/pysersic/assets/8907740/a8bfd341-9e02-4301-a61c-7c21839d48a7">

<img width="1325" alt="Screenshot 2024-06-06 at 12 50 46 PM" src="https://github.com/pysersic/pysersic/assets/8907740/ca72ae0d-98d2-4a06-80ad-b068c24fafb4">

**Comment by tbmiller-astro** [comment]
Thanks! That's interesting I've never seen it off center actually. The few times I have witnessed it radiates from the galaxies center only.  That is strange.

**Comment by prappleizer** [comment]
Yeah, So the center of galaxy case is more common; I have those as well. i had jotted these down for being EXTRA weird for having two.

The following are more common, which tend to be a single banding that emanates from the center. Already I'm noticing that it seems to happen with very edge on galaxies more often? And also perhaps tracks where a sharp feature exists on top of a elongated structure. The secondary ones might be due to additional features, or perhaps is some harmonic of the central one. 

<img width="1214" alt="Screenshot 2024-06-06 at 1 05 44 PM" src="https://github.com/pysersic/pysersic/assets/8907740/e4606700-aabd-476f-912b-34a981ffbd6d">


<img width="1210" alt="Screenshot 2024-06-06 at 1 04 45 PM" src="https://github.com/pysersic/pysersic/assets/8907740/6942e971-6b62-4ba9-b7a9-09a487fc3122">
 
<img width="1219" alt="Screenshot 2024-06-06 at 1 07 45 PM" src="https://github.com/pysersic/pysersic/assets/8907740/f4baed4d-76f6-4a9e-bdd4-573dc9d52978">
